tests/providers/apache/hdfs/hooks/test_webhdfs.py

```diff
@@ -85,61 +84,39 @@ def test_get_conn_with_schema(self, socket_mock, mock_get_connection, mock_insec
         mock_insecure_client.return_value.status.assert_called_once_with("/")
         assert conn == mock_insecure_client.return_value
 
-    @patch("airflow.providers.apache.hdfs.hooks.webhdfs.requests.Session", return_value="session")
-    @patch("airflow.providers.apache.hdfs.hooks.webhdfs.InsecureClient")
-    @patch(
-        "airflow.providers.apache.hdfs.hooks.webhdfs.WebHDFSHook.get_connection",
-        return_value=Connection(host="host_1.com,host_2.com", login="user"),
+    @pytest.mark.parametrize(
+        "test_connection",
+        [
+            pytest.param(Connection(host="host_1.com,host_2.com", login="user"), id="without-password"),
+            pytest.param(
+                Connection(host="host_1.com,host_2.com", login="user", password="password"),
+                id="with-password",
+            ),
+        ],
```

Comment:
Merge two identical test implementation (`test_get_conn_without_port_schema` and `test_get_conn_with_password_without_port_schema`)  into single one